www/src/utils/styles/index.js

```diff
@@ -60,6 +60,10 @@ export const buttonStyles = t => {
       color: `button.secondaryText`,
       fontWeight: `body`,
     },
+    disabled: {
+      pointerEvents: `none`,
```

Comment:
While this looks good, perhaps we should keep it consistent with 

```
cursor: `not-allowed`,
opacity: `0.5`,
```

(that's from line 129 in the same file)

What do you think?

Reply:
Sure, it makes a lot of sense to keep it consistent! I'll prepare a commit with the feedback changes.

Reply:
You're awesome!

Reply:
Let's remove ``pointerEvents: `none`,`` since it seems unclear what value that brings?

Reply:
@sidharthachatterjee it's necessary in here, because otherwise, the user will be able to click on the link. Because under `Button` component used in this view, we use `Link` one from gatsby, it can be only disabled by `pointerEvents: `none` css prop.